statistics.rb

```diff
@@ -0,0 +1,11 @@
+def top_reader
+    orders_grouped = orders.group_by(&:reader)
+    tab = orders_grouped.max_by { |_entity, reader| reader.count }.first
+    puts "#{tab} is the top reader"
+  end
+
+  def top_book
+    orders_grouped = orders.group_by(&:book)
+    tab = orders_grouped.max_by { |_entity, book| book.count }.first
+    puts "#{tab} is the top book"
+  end
```

Comment:
once again, if you moved logic to other file, remove this crap out of pull request.